src/test/java/gov/usgs/wma/waterdata/collections/BaseCollectionsIT.java

```diff
@@ -44,8 +44,14 @@ protected String doCollectionRequest(String path, HttpStatus expectedStatus) {
 	}
 
 	protected void doJsonCompare(ResponseEntity<String> rtn, String compareFile) {
+		doJsonCompare(rtn, compareFile, "");
+	}
+
+	protected void doJsonCompare(ResponseEntity<String> rtn, String compareFile, String filterOptions) {
 		try {
-			JSONObject featureCollection = new JSONObject(getCompareFile(compareFile));
+			String compareJson = getCompareFile(compareFile);
+			compareJson = compareJson.replace("{FilterOptions}", filterOptions);
```

Comment:
Ah yes, here is the replacement of the filter options that I asked about below.

Reply:
we needed this fix to allow the test to pass in filerOptions and use the doCompare function